frontend/src/App.tsx

```diff
@@ -35,7 +39,12 @@ export default function App() {
             <Route path=":cmoPatientId" />
           </Route>
           <Route path="/samples" element={<SamplesPage />} />
-          <Route path="/cohorts/" element={<CohortsPage />}>
+          <Route
+            path="/cohorts/"
+            element={
+              <CohortsPage userEmail={userEmail} setUserEmail={setUserEmail} />
+            }
+          >
```

Comment:
Passing the `userEmail` state value and state setter to Cohorts page for the `billedBy` field.